ui/server/src/test/scala/pl/touk/nussknacker/ui/api/AppResourcesSpec.scala

```diff
@@ -98,25 +90,6 @@ class AppResourcesSpec extends FunSuite with ScalatestRouteTest
     }
   }
 
-  test("it should return global config") {
```

Comment:
Why u removed that?

Reply:
It was moved to  test("it should return buildInfo"). And this test doesn't check global config.